### ISSUE TITLE
AUTHENTICATION section's SET button does nothing for Bearer

### ISSUE THREAD

**Issue by elonmallin** [mrin9/RapiDoc#27]
The "SET" button in the "AUTHENTICATION" section does nothing when the security scheme is bearer:
```
...
"security": [
    {
      "Bearer": []
    }
  ]
...
"components": {
  "securitySchemes": {
        "Bearer": {
          "type": "http",
          "scheme": "bearer",
          "bearerFormat": "JWT",
          "description": "Some description"
        }
      }
}
```

If I use `"type": "apiKey"` and set the `name` and `in` fields the "SET" button will work though.

**Comment by elonmallin** [comment]
Found the problem at: https://github.com/mrin9/RapiDoc/blob/master/src/components/security-schemes.js#L149
Seems like `inputname` is never set for `http/bearer` but it is for `apiKey`. Not sure exactly where to solve it though.

Could simply apend `|| apiEl.querySelector('input')` to make sure it gets the input if we don't find it by `inputname`.

**Comment by mrin9** [comment]
thanks for pointing it out, will rework on this piece to fix this and another [issue](https://github.com/mrin9/RapiDoc/issues/21)

**Comment by mrin9** [comment]
This should be fixed in 5.0.3
